src/scss/global.scss

```diff
@@ -41,6 +41,14 @@ a {
   }
 }
 
+button,
+select,
+[role='switch'] {
```

Comment:
Would it be possible to consolidate the focus styles a bit, and pull the anchor focus style from https://github.com/COVID19Tracking/website/blob/a7aa3c1ae1674b421b3fde601c5715bd3aba3e9b/src/scss/global.scss#L39-L41 into this selector as well?

(Minor suggestion, not a blocker for merge.)

Reply:
@beep I looked into this, but the `a { }` contains a lot of other styles that shouldn't be applied to these other elements.

I could do a placeholder just for the focus styling to DRY things up, though?

Reply:
@vijaysharwar Oh sorry! My suggestion was to _just_ move the `:focus` selector out of the `a { … }` block, and include an `a` selector in your new rule. So it’d read

```css
a {
  color: $color-link;
  &:active {
    color: $color-link--active;
  }
  &:hover,
  &:focus {
    color: $color-link--hover;
    text-decoration: none;
  }
}

a,
button,
[role="switch"] {
  &:focus {
    outline: 2px dotted; 
  }
}```

Does that make more sense?

(Again, this is _totally_ a minor point; doesn’t need to hold up merging!)

Reply:
Or do you mean something like 5fd511f?

Reply:
I mean something _exactly_ like https://github.com/COVID19Tracking/website/commit/5fd511feb01556a9469bad5a4982f8399fefac90! Looks fantastic!

Thanks, @vijaysharwar!